esignet-service/internal/metrics/metrics.go

```
@@ -0,0 +1,78 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package metrics
```

golangci-lint (2.12.2)

[warning] package-comments: should have a package comment

(revive)